features/smells/subclassed_from_core_class.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Smell - SubclassedFromCoreClass
+  Scenario: Configure multiple directories
```

Comment:
That scenario title is highly misleading. That our configuration works across multiple directory is a feature that is independent from SmellDetectors. How about just "Reports smell"?
You can then also simplify the paths below, "web_app/app/..." is not necessary.